numpy/linalg/linalg.py

```diff
@@ -2357,9 +2357,7 @@ def multi_dot(arrays): return functools.reduce(np.dot, arrays)
     >>> # or
     >>> A.dot(B).dot(C).dot(D)
 
-
-    Example: multiplication costs of different parenthesizations
-    ------------------------------------------------------------
+    Another example: multiplication costs of different parenthesizations.
```

Comment:
Not really an example at all - perhaps notes, or more details or something. Perhaps a third-level heading (`~~~`) under notes instead?